### PR TITLE
Marauders Rocket Dash

### PR DESCRIPTION
**Pull request by DeityLink**
This is something that I've thought about adding ever since I gave mechas the ability to crash through windows and tables when sent from a mass driver. A short-cooldown, short-range dash for marauders.

rocket_dash

As it stands, Marauders are a liability to deathsquadies, as their armors are usually already enough to protect them from most threats, and these mechas are so slow. I decided to make them a bit more fun to use at least.

These dashes also let them stun mobs and break tables or windows. They don't stun already lying mobs.

roll_over

window_breaker

Functionally, it works just like being sent out from a Mass Driver, but with a short range indoors. So if you send a Marauder from a Mass Driver it'll activate its flames for added coolness.

This also means that they remain active for the whole duration of a trip through space. The power consumption however is only taken into account when the ability is triggered.

space

And lastly here are the versions for the Seraph and Mauler.

colours

:cl:
* rscadd: Marauders, Seraph and Maulers can now perform rocket dashes. Let's them throw themselves forward without the need for a Mass Driver, though it has a short range indoors.

### CODE REVIEW

**Comment by Zennyization** [comment]
Love it.

**Comment by ghost** [comment]
Adorkable and badly needed/10

**Comment by yclatious** [comment]
10/10, merge when

**Comment by Blithering** [comment]
This is awesome, shame it'll rarely be seen. Does this replace the current marauder ability to zoom out your camera, or is it on top of it? Also does the mech launch bay at centcomm work, because this seems perfect to launch for a mission

**Comment by DeityLink** [comment]
@Blithering on top of it. Also I am temporarily blocking the mech launch bays at Centcom in #15480 because I have plans to improve them, but I agree with your statement, that was part of my plans.

File: code/game/mecha/combat/marauder.dm [hunk 1]
```diff
@@ -191,6 +247,8 @@
 	output += {"<b>Smoke:</b> [smoke]
 					<br>
 					<b>Thrusters:</b> [thrusters?"on":"off"]
+					<br>
+					<b>Roclet-Dash:</b> [dash_ready?"ready":"recharging"]
```
Comment (Intigracy) [inline code/game/mecha/combat/marauder.dm:251]: >Roclet

**Comment by Intigracy** [comment]
Is there logging for hitting someone wtih a dash?

Does dashing deal damage?

**Comment by DeityLink** [comment]
@Intigracy 5 brute damage. Can't hurt (or affect) people lying down.

**Pull request merged**